libs/langchain-google-common/src/utils/media_core.ts

```diff
@@ -0,0 +1,465 @@
+import { v4 as uuidv4 } from "uuid";
+import { BaseStore } from "@langchain/core/stores";
+import { Serializable } from "@langchain/core/load/serializable";
+
+export interface MediaBlobParameters {
+  data?: Blob;
+
+  metadata?: Record<string, unknown>;
+
+  path?: string;
+}
+
+/**
+ * Represents a chunk of data that can be identified by the path where the
+ * data is (or will be) located, along with optional metadata about the data.
+ */
+export class MediaBlob
+  extends Serializable   // FIXME - I'm not sure this serializes or deserializes correctly
+  implements MediaBlobParameters
+{
+  lc_serializable = true;
+
+  lc_namespace = ["langchain", "google-common"]; // FIXME - What should this be? And why?
```

Comment:
Only applicable if we want it to show up in traces (which we maybe do?) and just needs to match a Python equivalent

Reply:
All of the stuff that I currently have in `media_core.js` are things that I think would make sense to be in `langchain/core` at some point. (ie - they aren't Google specific)

So perhaps wait till @eyurtsev weighs in on how to set the namespace for them.